data/test-data/corsikaConfigTest.yml

```diff
@@ -0,0 +1,44 @@
+# RUn number
+nrun: 10
+
+# Number of showers per run
+nshow: 100
+
+# Primary particle name - proton, gamma, ...
+primary: 'proton'
+
+# Energy range - [emin, emax]
+erange:
+- !astropy.units.Quantity
+  unit: !astropy.units.Unit {unit: GeV}
+  value: 0.01
+- !astropy.units.Quantity
+  unit: !astropy.units.Unit {unit: GeV}
+  value: 10.0
+
+# Energy slope
```

Comment:
is 'energy slope' the right expression? Better to use what is used in the corsika manual 'slope of  energy spectrum'

Reply:
Fixed.